.github/actions/deploy/deploy.py

```
@@ -0,0 +1,1246 @@
+
+        # Path to the ConfigMap file
+        configmap_file = './.github/actions/deploy/openshift-service-ca-cert-configmap.yaml'
+
+        try:
+            # Apply the ConfigMap to the deployment namespace
+            self.run_command([
+                'kubectl', 'apply', '-f', configmap_file, '-n',
+                self.deployment_namespace
+            ])
+            print('✅ Created openshift-service-ca ConfigMap successfully')
+        except Exception as e:
```

Ruff (0.14.10)

Do not catch blind exception: `Exception`

(BLE001)